src/components/ViewWrapper.vue

```diff
@@ -324,8 +329,8 @@ export default class App extends Vue {
 
     let filingComplete: any
     try {
-      let filing: AlterationFilingIF | ChgRegistrationFilingIF | ConversionFilingIF |
-      RestorationFilingIF | SpecialResolutionFilingIF
+      let filing = null as AlterationFilingIF | ChgRegistrationFilingIF | ConversionFilingIF |
```

Comment:
This is so it has a value of null (instead of undefined) if all the if statements below are false.

Yes, both null and undefined are "falsy", but null is reactive whereas undefined is not, so this seems like a good practise.